### PR TITLE
[common] Add more information to README.md file

### PR DESCRIPTION
**Pull request by mykola-kobets-epam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,147 @@
 [![CI](https://github.com/aoscloud/aos_core_iam_cpp/workflows/Build/badge.svg)](https://github.com/aoscloud/aos_core_iam_cpp/actions?query=workflow%3ABuild)
 [![codecov](https://codecov.io/github/aoscloud/aos_core_iam_cpp/branch/main/graph/badge.svg?token=kg8h7ATd9S)](https://codecov.io/github/aoscloud/aos_core_iam_cpp)
 
-# aos_core_iam_cpp
+# Aos core IAM manager
```
Comment (al1img) [inline README.md:4]: IA manager (identity and access)

File: README.md [hunk 2]
```diff
@@ -1,4 +1,147 @@
...
+
+## Prepare build environment
+
+```sh
+sudo apt install lcov libsofthsm2 libsystemd-dev
+pip install conan
+```
+
+# Build for host
```
Comment (al1img) [inline README.md:13]: Should be only one main header. Please install `markdownlint` for VSCode and check.

File: README.md [hunk 3]
```diff
@@ -1,4 +1,147 @@
...
+
+A simple script is created to build a manager for the host system
```
Comment (al1img) [inline README.md:15]: add : at the end

File: README.md [hunk 4]
```diff
@@ -1,4 +1,147 @@
...
+| `CMAKE_BUILD_TYPE` | `Release`, `Debug`, `RelWithDebInfo`, `MinSizeRel` |
+| `CMAKE_INSTALL_PREFIX` | overrides default install path |
+
+## Run unit tests
+
+Configure example:
+
+```sh
+cd ${BUILD_DIR}
+conan install ../conan/ --output-folder . --settings=build_type=Debug --build=missing
```
Comment (al1img) [inline README.md:46]: Is it required if we use `host_build.sh`?

Comment (mykola-kobets-epam) [reply README.md:46]: no, it is required when cmake options are different from the ones in ./host_build.sh

Comment (al1img) [reply README.md:46]: Then it should be stated that it is required if you need to change options. And worth to mention which options are set by ./host_build.sh

**Review by al1img — APPROVED** [review]

**Review by MykolaSolyanko — APPROVED** [review]

**Review by mlohvynenko — APPROVED** [review]

**Pull request merged**